src/frontend/src/App.vue

```diff
@@ -1,29 +1,19 @@
 <template>
   <div id="app">
     <div class="main">
-      <div class="main__wrapper">
-        <div class="main__header">
-          <img
-            src="@/assets/img/logo.svg"
-            width="300"
-            height="47"
-            alt="V!U!E! Pizza"
-          />
-        </div>
-        <h1>Добро пожаловать!</h1>
-        <p>
-          Это проект V!U!E! Pizza для обучения на профессиональном
-          онлайн‑курсе<br />
-          <b>«Vue.js для опытных разработчиков».</b>
-        </p>
-      </div>
+      <mainPage></mainPage>
     </div>
   </div>
 </template>
 
 <script>
+import mainPage from "./views/Idnex";
 export default {
   name: "App",
+
+  components: {
+    mainPage,
+  },
```

Comment:
1. Ошибка Idnex вместо Index
2. Имена компонентов правильнее будет писать с большой буквы
```suggestion
import Index from "./views/Index";
export default {
  name: "App",

  components: {
    Index,
  },
```